src/RapidezServiceProvider.php

```diff
@@ -285,6 +262,19 @@ protected function registerConfigs(): self
             $this->mergeConfigFrom(__DIR__ . '/../config/rapidez/' . $configFile . '.php', 'rapidez.' . $configFile);
         }
 
+        // Find all store-specific config files and merge them in
+        // TODO: Add how to use this in the Rapidez documentation
+        foreach (glob(__DIR__ . '../config/rapidez/stores/*/*.php') as $configFile) {
```

Comment:
Isn't this a heavy task? Does this work with the config caching?

Reply:
Hmm, we could cache the list of config files we get out of the glob and the few lines after? Keeps the code relatively simple and saves on the performance cost of that part.

Reply:
- Doesn't Laravel provide something for this?
- Maybe simply override the config part from Laravel and add a store check there?
- Or reverse it; currently we're checking all files but we do know the stores from the database, maybe use that? Or keep a "custom store config list" somewhere?